files/tasks.py

```
@@ -1339,18 +1339,72 @@ def cleanup_orphaned_uploads():
+            headers=headers,
+            timeout=10
+        )
+
+        if response.status_code == 200:
+            try:
+                response_data = response.json()
+            except (json.JSONDecodeError, ValueError):
+                logger.error(
+                    f"Newsletter subscription for {email}: failed to parse JSON response. "
+                    f"Response content: {response.text[:200]}"
+                )
```

Ruff (0.14.6)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)